kmcd-props.md

```diff
@@ -132,6 +135,18 @@ Total dai of all users = CDP debt for all users and gem + system debt (vice)
     rule sumOfAllUserDebt(.Map, _, SUM) => SUM
 ```
 
+Total backed debt (sum over each CDP's art times corresponding ilk's rate)
+
+```k
+    syntax Rat ::= calcSumOfScaledArts(Map, Map) [function]
+                   | calcSumOfScaledArtsAux(List, Map, Map, Rat) [function]
+ // ------------------------------------------------------------------------------------
```

Comment:
formatting (line up the `|` with the `=` on previous line, shorten the `---`)

Reply:
Done.